### PR TITLE
Airtable refactor

### PR DESCRIPTION
**Pull request by 0rnella**
## Changes:
- Deleted the postgres setup (since we have this commit history, we could always go back to it later through the log)
- Added environment variable setup (through an npm script): these variables are set in each environment that the server is running on, and they are a good way to obscure things like the Airtable Key
- Added instructions about the setup in general, and the env stuff specifically, in the README
- I also added a few files in a apiMethods folder:
1. the base.js file is to get the info of our Airtable base, and instantiate the `base` variable that we'll keep using throughout
2. the companies.js file is to have all functions that will query things related to companies
3. the index.js will put together all the necessary functions from all the different files in the folder and re-export them

## Note
These aren't super straightforward so let me know if you need a walkthru, or feel free to leave comments and I'll answer!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
+.env
```
Comment (0rnella) [inline .gitignore:3]: this means the .env file will never be committed, and will stay in your local setup only

File: apiMethods/base.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const Airtable = require('airtable');
+require('dotenv').config();
+
+const apiKey = process.env.AIRTABLE_KEY;
```
Comment (0rnella) [inline apiMethods/base.js:4]: this is where we use the AIRTABLE_KEY that is set in the .env file

File: package.json [hunk 3]
```diff
@@ -5,9 +5,9 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "start": "nodemon index.js",
+    "start-dev": "NODE_ENV=development nodemon index.js",
     "startdb": "node database/database_setup.js",
-    "test": "test"
+    "env-setup": "touch .env & echo 'AIRTABLE_KEY=' >.env"
```
Comment (0rnella) [inline package.json:10]: this is the script that creates a new file  (`touch .env`) and then creates some initial content inside of that file (`echo 'AIRTABLE_KEY=' >.env`). let me know if you need more explanation!

File: package.json [hunk 4]
```diff
@@ -21,6 +21,7 @@
   "homepage": "https://github.com/0rnella/vibe_check#readme",
   "dependencies": {
     "airtable": "^0.9.0",
+    "dotenv": "^8.2.0",
```
Comment (0rnella) [inline package.json:24]: this is the library that allows us to easily load the environment variables

**Review by pyuli15 — APPROVED** [review]

**Pull request merged**